src/foundry/common/utils/helpers.d.mts

```diff
@@ -393,6 +404,33 @@ export interface MergeObjectOptions {
   performDeletions?: boolean | undefined; //TODO: implement this in the mergeObject return type
 }
 
+/**
+ * A helper function for merging objects when the target key does not exist in the original
+ * @internal
+ */
+declare function _mergeInsert(
+  original: object,
+  k: string | number | symbol,
+  v: any,
```

Comment:
If I had to guess this'd also be `object` but they're an `@internal` function so it's less important to get typings of these exactly right.

Reply:
From what I read, `object` is distinct from base types, though, right? And you can insert strings or numbers with this.

Reply:
Ah. Then `any` might be fine. Is there anything that's NOT allowed like a class?

Reply:
I don't think so. Although in spirit, this is only used by `mergeObject` , which I don't think usually merges things with classes in it. But you never know. There is no such restriction in theory.

Reply:
Is this the kind of "merging with classes" you're referring to?

![image](https://github.com/League-of-Foundry-Developers/foundry-vtt-types/assets/16617088/1b319347-c97b-4067-aa2b-6483d8c96736)